packages/frontend/src/components/login/v2/NetworkFeeAllowance.js

```diff
@@ -26,8 +40,14 @@ const StyledContainer = styled.div`
     }
 `;
 
-export default () => (
+export default ({ contractId, contractIdUrl }) => (
```

Comment:
Not a blocker, but we should probably rename this thing to e.g. `LoginDetails` now that it's not really specific to the  Network Fee Allowance...